### PR TITLE
Add composer.json

### PR DESCRIPTION
**Pull request by vstm**
This adds support for composer, without addressing the PSR-0 issue #3 directly. Refactoring the code to PSR-0 (or better PSR-4 since PSR-0 is deprecated) would break backwards-compatibility.

This solution uses a classmap so we don't have to introduce any changes to the code and the package can still be used with composer.

Things to do:
- Mabe discus the vendor prefix. I have just used `singpolyma` which is probably ok. The full package name would then be `singpolyma/openpgp-php`.
- Submit it to packagist so it's easier to use with composer. This can be done as soon as the composer.json is in the official repository.

### CODE REVIEW

File: tests/bootstrap.php [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?php
+@include_once dirname(__FILE__) . '/../vendor/autoload.php';
```
Comment (singpolyma) [inline tests/bootstrap.php]: Need a newline at the end of this file at the least.

What does this and the change to `phpunit.xml` accomplish?  Is autoload generated by composer?  I'd rather not be dependent on composer just to run the tests if we can manage it.

**Comment by singpolyma** [comment]
Thanks for working on this!

File: tests/bootstrap.php (hunk 1 above)
Comment (vstm) [inline tests/bootstrap.php]: @singpolyma thank you for the feedback. Correct, the `vendor/autoload.php` is generated by composer and it sets up [the PHP class autoloader](http://php.net/manual/en/language.oop5.autoload.php) and the include path (in this case for phpseclib).

Without composer the `include_once` would not do anything and with the `@` in front of it, there would be no warnings about the missing file (I could also do a file_exists first). So it doesn't add a dependency on composer - it is just included if it's there. 

But I admit the bootstrap thing isn't really required for people who want to just load openpgp-php as a dependency to their project - they just need the composer.json. The bootstrap-change would just make things easier for developers who actually work on openpgp-php itself and are using composer to resolve the phpseclib-dependency.

So if you think that this change is too much - which I can fully understand - you could copy the composer.json into the project root and close this PR (or I can do a new PR with just the composer.json in it).

If I was able to convince you that this doesn't force a composer dependency and you are ok with the bootstrap-thing, I will of course add a blank line at the end (sorry for missing this).

Comment (singpolyma) [inline tests/bootstrap.php]: Ok, that all makes sense. Add the newline and I will merge.

Comment (vstm) [inline tests/bootstrap.php]: Thanks, it's done!

**Pull request merged**